src/app/home-page/dashboard/dashboard.component.html

```diff
@@ -19,7 +19,7 @@
         <app-mastodon-timeline fxFlex.md="35" fxFlex.gt-md="35" fxFlex.lt-md="100"></app-mastodon-timeline>
       </div>
       <div class="w-100">
-        <div class="mat-card-title text-align-center">Featured Content</div>
```

Comment:
This no longer worked because we were applying an internal Material class to a div and this class no longer exists. Instead of using the new class name, I made this an `h3` instead so it's less likely to break in the future